### ISSUE TITLE
Will morerefinedstorage keep chunkloading?

### ISSUE THREAD

**Issue by 200542053** [tth05/morerefinedstorage#41]
#### Issue description:

Will morerefinedstorage keep chunkloading?
I am the only one in the server, but spark and LagGoggles shows, the refinedstorage's thread is always running, is it possible to add an option in the config for turning off chunkloading?

#### Version (make sure you are on the latest version before reporting):

- Minecraft: 1.12.2
- Forge: 2680
- Refined Storage: 2.3.1

Does this issue occur on a server? [yes]

#### If a (crash)log is relevant for this issue, link it here:

[pastebin/gist/etc link here]

**Comment by tth05** [comment]
Yes, currently there are a few places where the system will force load chunks. I remember some commits in the upstream repository about this, which could probably be backported to disable chunk loading by refined storage in all places.
